python/enthic/compute_stats.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+"""
+====================================================================
+Compute statistics from companies's data after complete and check it
+====================================================================
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import math
+from enthic.arbre_compte_de_resultat import TREE_VIEW
+
```

Comment:
PEP 8: E302 expected 2 blank lines, found 1, use autopep 8 or an IDE that reformat according to PEP8 e.g. Pycharm Ctrl+Alt+L .